lib/school.rb

```diff
@@ -0,0 +1,27 @@
+class School
+  attr_reader :start_time,
+              :hours_in_school_day,
+              :student_names
+
+  def initialize(start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+  end
+
+  def add_student_name(name)
+    student_names << name
+  end
+
+  def end_time
+    @start_time += @hours_in_school_day
+  end
+
+  def is_full_time(hours_in_school_day)
```

Comment:
Be careful to build _exactly_ what we ask for - we wanted this method to be `is_full_time?` with no arguments